docs/getting-started/core-concepts/plugins.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "PluginDefinitions and Plugins"
+weight: 4
+---
+
+## What are Plugin Definitions?
+
+PluginDefinitions are a way to extend the functionality of the platform. They can include a [Juno microfrontend](https://github.com/cloudoperators/juno) into the Greenhouse Dashboard and/or a backend component. The backend components of a PluginDefinition is packaged as a Helm Chart and provides default values for the Helm Chart. This allows to package and distribute tools such as Prometheus with a sensible default configuration, as well as giving the user a list of configurable values.
+
+## Plugins
+
+In order to deploy the Helm Chart of a Plugin Definition into a particular cluster a Plugin is used. The Plugin specifies the PluginDefintion to deploy, the release namespace to use, the values to set and the cluster to deploy to. Depending on the PluginDefinition it may be required to provide required values (e.g. credentials, endpoints etc.), but in general the PluginDefinition provides sensible defaults.
```

Comment:
```suggestion
A Plugin is used to deploy a Helm release described in a PluginDefinition. The Plugin specifies the PluginDefinition to be installed, the option values to be set as well as the cluster and namespace to be deployed to. Depending on the PluginDefinition, it may be necessary to specify required values (e.g. credentials, endpoints, etc.), but in general the PluginDefinition provides well-established default values.
```